src/sink.rs

```diff
@@ -99,6 +100,18 @@ impl Sink {
         self.pause.load(Ordering::SeqCst)
     }
 
+    /// Stops playback.  To resume, a new sound must be added with `append()`
+    pub fn stop(&self) {
+        self.stopped.store(true, Ordering::Relaxed);
+    }
+
+    /// Gets whether a sound is stopped.
+    ///
+    /// A stopped sound cannot be resumed, a new one must be started with `append()`.
+    pub fn is_stopped(&self) -> bool {
```

Comment:
Reasonable enough.  Do you want me to change the behavior of `is_stopped()` or just remove it altogether?